### ISSUE TITLE
Personal do not mean what you think...

### ISSUE THREAD

**Issue by deejam** [KapiX/Koder#101]
While translating Koder, I came across this sentence: "Something wrong has happened while opening the configuration file. Your personal settings will not be %s%."

Personal is related to "person", which is an organization.

"Your" suggest the message is for who controls the user account. Whatever controlling the active user (usually a human being, but sometimes a computer or another type of machine) is an individual, not a person.

The settings belong to the user account, which also isn't an organization.

Individual means "single, unique" which is what I think is the intended meaning.

"personal" should be replaced with "individual".

**Comment by waddlesplash** [comment]
Merriam-Webster is of course correct, "person" in the singular refers to a single individual human, and in the plural "persons" to a group of people without any other implications; certainly I wouldn't infer they are part of any common organization from the usage of "persons" alone.

"Personal settings" makes sense here, it has the same meaning as "personal preferences" (in a technical context, the two can mean separate things outside of a technical contexts). The latter is a very common phrase in colloquial English apart from computing.

So ... I'm pretty confused as to what the issue is, also.

**Comment by KevinAdams05** [comment]
I'm also a native English speaker and agree with Waddlesplash, "Personal" is being used correctly.

**Comment by deejam** [comment]
> Merriam-Webster is of course correct, "person" in the singular refers to a single individual human, 
What you describe is in law a NATURAL PERSON, not a PERSON.
>Definition of NATURAL PERSON: A human being, **naturally born**, versus a legally generated juridical.
Compared to a JURIDICIAL PERSON (or LEGAL PERSON]:
>Definition of JURIDICAL PERSON: Entity, as a firm, that is not a single natural person, as a human being, authorized by law with **duties and rights**, recognized as a legal authority having a distinct identity, a legal personality. Also known as artificial person, juridical entity, juristic person, or legal person. Also refer to body corporate.
Note that a JURIDICIAL PERSON har **duties and rights** (meaning, it is not free)...
...as does a PERSON:
>Definition of PERSON: A man considered according to the rank he holds in society, with all the **rights** to which the place he holds entitles him, and the **duties** which it imposes. 1 Bouv. Inst. no. 137. A human being considered as capable of having **rights and of being charged with duties**; while a “thing” is the object over which rights may be exercised.

>and in the plural "persons" to a group of people without any other implications; certainly I wouldn't infer they are part of any common organization from the usage of "persons" alone.
In English, plural of PERSON is PEOPLE; a group of **natural beings** thay may share common culture or traditions under COMMON LAW (but the dictionary will not explain the difference between English and Legalese, and when which is used).
In LAW, plural of PERSON is PERSONS; **artificial persons** that are members of a state, bound by statues under CIVIL LAW.
> Persons would be used as a the plural of person when you are not referring to a group collectively but to a **collection of individuals**. For example, "Many people like ice cream," but "Corporations are considered persons under the U.S. Consitution."

Personal do not mean what you think.

TLDR: NATURL PERSON is a human being. PERSON is a fictive entity.

**Comment by KevinAdams05** [comment]
> What you describe is in law a NATURAL PERSON, not a PERSON.

Legal definitions of words often differ from the colloquial definition. This is irrelevant since this is not a legal case. In normal usage the distinction between natural person and person is not made. 

Personal has referred to Person for centuries and major dictionaries support that. 

This ticket was closed, there is no point in debating any further. 

Webster:

> Definition of personal
> 1 : of, relating to, or affecting a particular **person** : private, individual 
> _personal ambition_
> _personal financial gain_
> 3 : relating to the **person** or body
> 4 : relating to an **individual** or an individual's character, conduct, motives, or private affairs often in an offensive manner a personal insult


Oxford:

> 1 Belonging to or affecting a **particular person** rather than anyone else.
> ‘her **personal** fortune was recently estimated at £37 million’
> 1.1 Done or made by a **particular perso**n; involving the actual presence or action of a particular individual. ‘the President and his wife made personal appearances for the re-election of the state governor’
> 2 Of or concerning **one's** private life, relationships, and emotions rather than one's career or public life.

**Comment by deejam** [comment]
> Legal definitions of words often differ from the colloquial definition.
Which is exactly my point; words that is used in conversation do not always mean what the majority think. 

> This is irrelevant since this is not a legal case.
Incorrect.

> In normal usage the distinction between natural person and person is not made.
What is normal usage? All types of contracts (written or not) that in some capacity is applicable in court will make a distinction between person/person/personal. 

> Personal has referred to Person for centuries and major dictionaries support that. 
I do not dispute that personal is related to person. I dispute the actual meaning of person and therefor the meaning of personal.

History show that persona (from which person is derived) has nothing to do with human beings:
> persona (from wiktionary):
>
> Etymology
>
> Borrowed from Latin persōna (“mask; character”), of uncertain origin. Possibly from personō (“to sound through”); or from Ancient Greek πρόσωπον (prósōpon, “face; appearance; mask used in ancient theatre to denote a character or, more generally, a social role”); or from Etruscan 𐌘𐌄𐌓𐌔𐌖 (φersu). 
>
> Noun
>
> persona (plural personas or personae or personæ)
>
 >   A social role.
 >   A character played by an actor.
 >   (psychology) The mask or appearance one presents to the world.
 >   (marketing, user experience) An imaginary person representing a particular type of client or customer, considered when designing products and services that will appeal to them.

**Comment by KapiX** [comment]
> words that is used in conversation do not always mean what the majority think.

If majority thinks that the word means something, then it means that. Language is a tool and how words are used determines their meaning. Languages (and words by extension) do evolve.

I'm locking this issue because everything than can be said has been said already.
I have no interest in changing the wording in this particular case.
